realsense2_camera/src/named_filter.cpp

```diff
@@ -263,3 +263,14 @@ void PointcloudFilter::Publish(rs2::points pc, const rclcpp::Time& t, const rs2:
             _pointcloud_publisher->publish(_msg_pointcloud);
     }
 }
+
+
+AlignDepthFilter::AlignDepthFilter(std::shared_ptr<rs2::filter> filter, 
```

Comment:
Shouldn't this be in its own .cpp? `named_filter.cpp` should be only for the `NamedFilter` class, no?